src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      return callback(value);
+    },
+
+    add(value) {
+      calculator.result += value;
```

Comment:
Instead of 'calculator.result', you should use 'this.result' to reference the current calculator object.